### DISCUSSION TITLE
streams-i2s-a2dp

### DISCUSSION THREAD

**Discussion by paralogicdr** [pschatzmann/arduino-audio-tools#1138] [Q&A] [answered]
While uploading the streams-i2s-a2dp example to the esp32 dev kit in the Arduino ide, I get the following error during the compile phase.

Sketch uses 1327537 bytes (101%) of program storage space. Maximum is 1310720 bytes.
Global variables use 45692 bytes (13%) of dynamic memory, leaving 281988 bytes for local variables. Maximum is 327680 bytes.

anyone has the same problem? how can I solve this?

**Comment by aagw** [comment, marked as answer]
Hello
Try change your partition scheme to: Huge APP in the tools menu in the Arduino IDE.

**Comment by paralogicdr** [comment]
thank you, it worked
